src/config.rs

```diff
@@ -42,6 +55,9 @@ pub struct Server {
     /// Channels to automatically join. Any `/join` commands in `auto_cmds` will be moved here.
     #[serde(default)]
     pub join: Vec<String>,
+
+    /// Authenication method
+    pub auth: Option<Auth>,
```

Comment:
After renaming `Auth` to `SASLAuth` please also rename this field to `sasl_auth`.

Reply:
renamed.